conf/modules.config

```diff
@@ -165,7 +165,7 @@ process {
     }
     withName: SVANALYZER_SVBENCHMARK {
         ext.prefix = {"${params.sample}"}
-        ext.args   = {"-normshift 0.3 –normdist 0.3 –normsizediff 0.3"}
+        ext.args   = {"-normshift $params.normshift –normdist $params.normdist –normsizediff $params.normsizediff -maxdist $params.maxdist"}
```

Comment:
This doesn't really follow the nf-core guideline of using as less parameters as possible. (You can give these params via a config, which is the expected behaviour). @maxulysse can you please confirm this? (just to be sure :grin:)

Reply:
Those parameters are kind of important for the pipeline itself, I think we need to align benchmark params with the tools. Some of the parameter arrangements itself change the methodology (-duo-to-ins). Result of the metrics highly depend on those parameters. That is why I think we should be controlling benchmark parameters using the main config (for defaults we especially align).

Reply:
As you pointed out, this can heavily change between certain files. Wouldn't it better to have those as optional samplesheet values?

Reply:
Can you show me an example of how?

Reply:
You can easily do it using these steps:
1. Add those values as values of the samplesheet in `assets/schema_input.json` (make sure to add the `meta` for these values and that it has sensible defaults
2. access the meta values in the config (similar to how you just did with the parameters)